Sources/Ink/Internal/Reader.swift

```diff
@@ -108,7 +108,14 @@ extension Reader {
 
         return string[startIndex..<currentIndex]
     }
-
+    
+    mutating func discardWhitespaces() {
+        while !didReachEnd {
+            guard !currentCharacter.isNewline && currentCharacter.isWhitespace else { return }
```

Comment:
There is a `character.isSameLineWhitespace` convenience API that you can use here.

Reply:
Premature optimization. I do not want to remove the newline so my code works and I have been debugging string escaping all day and using your extension broke my test case edits?